.travis.yml

```diff
@@ -0,0 +1,89 @@
+dist: trusty
+
+sudo: required
+
+language: java
+
+jdk:
+- openjdk8
+- oraclejdk8
+
+services:
+- docker
+
+notifications:
+  email:
+    recipients:
+    - arfathpasha@gmail.com
+    on_success: never
+    on_failure: always
+
+env:
+  global:
+  - MONGODB_VER=mongodb-linux-x86_64-2.6.12 ANT_TEST=test WIRED_TIGER=false
+  - JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+  # TEST_TOKEN
+  - secure: tguBGgSgTvVY/ayL0IFA08JM8Nu0ys0iwU2nte2oZIl8J6db+8FwzV6SlXeRjYYJYYHu3uOobYb2i1OpEGfRH2aE8+Q0TZzuupSlKb2cEReV9SAL4w0cb0azg06gjKmY+NoA18lCDZKV7X15Zh9cGkRvcM0Mjbhbbdek3n8vf1h5F1Gp6sV7MHSwQEGRmvmyEy/qTb7D1qIVEyZaxWV+ou3kd4eagiaJr0P+rvqBUOvoqWsUrroR3Vfs2A5J5gy4IeZJhpxwJsJz3UapuWcmYFV+fU/DgGdx3oaG7O5+rBH/v4uzZGaha7xuLFP93UURpxPbQyXKhsE1OiRW+9svpZw3yAS5/PN49GIWcqg3vNECkuyPBO9iSDLzz6iUNiErlsePFKKIlk/qzwOJimmz1+8ZCeDHZXUv81JK+RhKK1dlWECZcpYrr4YsTV4GVkVORIcseIy2jLmvRIer0pLiauOPooiNVcCv+TiYkwfeZdO8QiwsxN1WBzjzniwS3cA6z/AKmm0qJNpu37JTO19AiKEqgcjEfbftSp7S/nnIdGxObFt84iToGb3XfdFVeI8O3B9ngFxuxILb0hYfaXm5TfD2WIUL8pL6g3GVVapFJDkal1PTmc6uq6jU8UW+3SirqRd34ELHcoOKI03cnVdaAgCbKViTdGZtjZjpuBkHz48=
+  # TEST_TOKEN2
+  - secure: uvWMPRCu9Ww4aZOwjaECspxJ3NLQwLOhn1p5ND0mUADupOqFHmG6aiejhD+FoQ4JETd0FofHUAOhtIzmZuKYj08f/v7q6/bipSXqEj1Ufb0hgS7V9AaItfsYvgEeXr0jj04C1rfHCliSZGyDW2MhNoeNjIKgmyD9EbJ3lqpuHI4zXmWUrPknqqkq9GIN+ZI8AEGN6P1FtVD5zfQ43poWAiVlPVUiT6q/wcuDANx4teYPIBWC4jMNZRAP44JzMkru/TIlgM5V9UgvscBAHVt6V5U7fHolQJ+chwFioQniTySKkA5PyLnnXeydKyUajEzrylhQgBq9NuHtUs4cqEiop5RTwQ4yDQdCK9/1gY9FNRMvdSj0Z0C7YwA+m6YA95tAONf8PGAL1bXAQfXZ58IRMqyf/Gu+PTuOGVX/NPbXqYgWBXl9HAi4Es8pXl0EX3m3OSDbfaiXd+7ogxdiYiWiSjKUGfHsNpCii7bg2Txec3QnqzXPR1T+Vd3kNHX6z2Ry83K5z1UBwHoBkfSLOULY4mxI6zbhXU0h2fadLUDeCiTAe9KTYOWq4xpxiepbWjg2VBnN3Lw4inlevan2iw/kh9ZQzEurZUo/PakIHPWmUn2RbG7QWu88LNh8Ku80ei1lfVwk/8IpT9V4Clx9FRm4zedE1dDdsXy5OhKGJCk0llM=
```

Comment:
secure: "UPzlmxopnwD86fGPuyd7VNt1TNGxd9IwhCbtusd8PopsxYW064uJ5Ddy8AgkzE4gms5Bb+CIGgG5w/+TukkU0MTim/iZg6us6nKKcQXYUyGeTJvGx7NuSAlKaQjfZGj6kEM/xLMFnv51XKXCzU0rdNDGIVL5mEZ9x4HvcFYIArKofPsUuh52MbsdzTy9xQszEdkigBuAX1fU0sZmpClyNSVDbiZJQmlC//z8a0jbaZAa66vhDLYK1bBDtbUvdwc8eiu6x1qKJ4lOdZAjr+oRRARJFVTckhz8lpXXExTnipFOYvLRI8woxtNMI4ZyZ2ALkr1bPz7UqJ+URlKFu4+fTxLFik/7QIGZB4Nn67oBsVTbUAESrfOqu2ihfYsRTNdkxgb6H4ubWXLjJHznIDkddLV7uX53awfd9fJ8Vik0ZjHDUwOR4xMHkux5nFZZD6hrXkXu7eg8/o6eYNpdwXylZMnf81KVYLn8KegLppIY3L5bI0rYCXrhuHcHDypI4Ir2YqE63NxWTmAg0KQW+R0d6CzMipVfbH1lwSR/eyjdPktFhjGGvehZAKE7KP+jnHG7Tdf/HTb/Jgvlggz1KqQSuouOGoqylKUJbI9fVrEpwrD3qKLc76fsSXBmE2J0EHW9mSEwODwgx5S8P17I8xUMYAnS0jAE51ErBS7zJ6AZ3s0="

Reply:
@sychan yes the encrypted "secure" tokens are kbase tokens. I realized from your comment that I should have also encrypted the codecov token as well. 
There is already a kbase account in codecov that we used for auth2. I am not sure who has access to this account. Here it is
https://codecov.io/gh/kbase/auth2

Reply:
@arfathpasha The requested changes are for travis encrypted vars that should work for this repo. Can you make the changes on your PR?